openapi/v2/restful/cluster.yaml

```diff
@@ -759,6 +759,16 @@ paths:
           in: query
           schema:
             type: string
+        - name: isFavorite
+          description: return user's favorite cluster
+          in: query
+          schema:
+            type: boolean
+        - name: withFavorite
```

Comment:
In what condition this field need to be used?

Reply:
list all cluster but should the favorite cluster